lib/VMwareWebService/broker_timeout.rb

```diff
@@ -0,0 +1,64 @@
+module Timeout
```

Comment:
I think we did decide that kill/raise is the problem. The modified timeout just avoids the case we were seeing. When kill/raise is fixed, we shouldn't need the timeout code change.